site/src/components/banner.md

```diff
@@ -0,0 +1,130 @@
+        </span>
+		<span>Title</span>
+	</p>
+	<p>A banner about something happening in the world <a href="#">Link Text</a></p>
+</section>
+```
+
+### Dismissing banner
+
+Banners can be dismissed by adding the `data-dismissable` attribute to the `.banner` and `data-dismiss`to the closing button.
+
+If `data-dismissable` attribute is also assigned a `dissapearing` value - banner would slowly slide up.
+
+**Note**: banner's content should be wrapped up with `banner-content` container which helps with correct positioning and spacing of content.
+
+```html-no-indent
+<section
+	class="banner"
+	data-uid="example-banner-uid"
+	data-dismissable="disappearing"
+>
+	<div class="banner-content">
+		<button
+			type="button"
+			data-bi-name="close"
+            data-dismiss
+			data-banner-dismiss="example-banner-uid"
+			class="button-reset button-close line-height-0 border-radius-rounded position-absolute top-0 right-0"
+		>
+			<span class="visually-hidden">Dismiss banner</span>
+			<span class="icon" aria-hidden="true">
+				<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 448">
+					<path
+						d="M269.254 224l137.373 137.373-45.254 45.254L224 269.254 86.627 406.627l-45.255-45.254L178.746 224 41.373 86.627l45.255-45.255L224 178.746 361.373 41.373l45.254 45.255L269.254 224z"
+						fill="currentColor"
+					/>
+				</svg>
+			</span>
+		</button>
+		<p>Dismissable banner. It would slowly slide up after closing.</p>
+	</div>
+</section>
+
+<section
+	class="banner margin-top-xs"
+	data-uid="example-banner-uid-2"
+	data-dismissable
+>
+	<div class="banner-content">
+		<button
+			type="button"
+			data-bi-name="close"
+            data-dismiss
+			data-banner-dismiss="example-banner-uid-2"
+			class="button-reset button-close line-height-0 border-radius-rounded position-absolute top-0 right-0"
+		>
+			<span class="visually-hidden">Dismiss banner</span>
+			<span class="icon" aria-hidden="true">
+				<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 448">
+					<path
+						d="M269.254 224l137.373 137.373-45.254 45.254L224 269.254 86.627 406.627l-45.255-45.254L178.746 224 41.373 86.627l45.255-45.255L224 178.746 361.373 41.373l45.254 45.255L269.254 224z"
+						fill="currentColor"
+					/>
+				</svg>
+			</span>
+		</button>
+		<p>Dismissable banner.</p>
+	</div>
+</section>
+```
+
+### Loading banner
+
+Ensure the user knows they need to wait for some event in order to interact with a banner.
+
+```html
+<section class="banner is-loading">
+	<p>Loading banner.</p>
+</section>
+```
+
+## Semantic colors
```

Comment:
@wibjorn Do you think we need different colors for banners? I usually see only purple ones, and leaning towards the idea of creating just one and if more would be needed - adding those in the future.